pages/docs/reference/comparison-to-java.md

```diff
@@ -21,11 +21,10 @@ Kotlin fixes a series of issues that Java suffers from:
 ## What Java has that Kotlin does not
 
 * [Checked exceptions](exceptions.html)
-* [Primitive types](basic-types.html) that are not classes
-* [Static members](classes.html)
-* [Non-private fields](properties.html)
-* [Wildcard-types](generics.html)
-* [Ternary-operator `a ? b : c`](control-flow.html#if-expression)
+* [Primitive types](basic-types.html) that are not classes - are used under the hood, but cannot be used explicitely. 
```

Comment:
[Primitive types](basic-types.html) that are not classes. The byte-code uses primitives where possible, but they are not explicitly available.